kolibri/core/assets/src/keen-config/variables.scss

```diff
@@ -1,3 +1,11 @@
-$brand-primary-color: #996189;
-$font-stack: 'Noto Sans';
+@import '~kolibri.styles.definitions';
+
+$brand-primary-color: $core-action-normal;
```

Comment:
What's "$core-action-normal"? It doesn't sound like a color code :)

Reply:
originally it was the normal color associated with UI elements that trigger an action; our color variables are admittedly in need of refactor though.